src/dev/ci_setup/setup_env.sh

```diff
@@ -53,6 +51,12 @@ export PARENT_DIR="$parentDir"
 kbnBranch="$(jq -r .branch "$KIBANA_DIR/package.json")"
 export KIBANA_PKG_BRANCH="$kbnBranch"
 
+###
+### Sets correct NODE_OPTIONS for CI
+###
+export KIBANA_CONFIG_DIR=${KIBANA_PATH_CONF:-"$KIBANA_DIR/config"}
+export NODE_OPTIONS="$(grep -v ^# < ${KIBANA_CONFIG_DIR}/node.ci.options | xargs) $NODE_OPTIONS"
```

Comment:
Why is this required? Won't `node.ci.options` just get picked up?

Reply:
We wanna expose those for every node process running on CI